inductiva/structures/simulators/fenicsx.py

```diff
@@ -0,0 +1,40 @@
+"""FEniCSx module of the API for finite element analysiss."""
+
+from typing import Optional, List
+from uuid import UUID
+
+from inductiva import types, tasks
+from inductiva.simulation import Simulator
+
+
+class FEniCSx(Simulator):
+    """Class to invoke a generic FEniCSx simulation on the API."""
+
+    def __init__(self, api_method: str = "fem.fenicsx.run_simulation"):
+        super().__init__()
+        self.api_method = api_method
+
+    @property
+    def api_method_name(self) -> str:
+        return self.api_method
+
+    def run(
+        self,
+        input_dir: types.Path,
+        commands: List[dict],
+        resource_pool_id: Optional[UUID] = None,
+        n_cores: int = 1,
```

Comment:
Do you know how to set the script you wrote on the backend to run in parallel, with a configurable number of cores?

Reply:
"I've never done something similar before. Do you have any examples I can follow?

Reply:
The way this is done varies with the simulator, I'd have to figure out how to do this in FEniCSx. Let's remove this parameter for now, and deal with this in the future, if/when needed.

Reply:
make sense, I will remove this parameter! thanks